9.hello-docker-compose/README.md

```diff
@@ -0,0 +1,130 @@
+# Dapr with Docker-Compose
```

Comment:
Unfortuntely sample 9 was already checked in so this will need to be called sample 10 for now.

Reply:
Updated to 10.